docs/community_features.md

```diff
@@ -330,8 +330,7 @@ Here is a list of features that have been added to the firmware as a list, group
 #### 4.1.3 - Fill Clips and Once Clips
 
 - ([#196] and [#1018]) In Song View (Rows mode), holding the status pad (mute pad) for a clip and pressing `SELECT`
-  brings up a clip type selection menu. For Grid Mode there is a dedicated Config Mode in which this menu can be
-  reached, see `Yellow Mode` in [Grid View](#415grid-view).
+  brings up a clip type selection menu. For Grid Mode, hold a clip and press `SELECT` to enter a `CLIP SETTINGS` menu where you can convert a clip to an audio clip and set the clip's launch style.
```

Comment:
Is this true at this point? I can't find the "convert a clip to an audio clip" option. I just see the launch style options.

Reply:
> Is this true at this point? I can't find the "convert a clip to an audio clip" option. I just see the launch style options.

Thanks that was outdated. I've updated the documentation now!